python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -163,15 +187,18 @@ def move_while_exposing(self, value):
         self.assert_corrections(enabled=False)
 
         # FIXME: Get position from telescope if elevation = 0.
-        azimuth = id_data.data.azimuth
+        azimuth = id_data.data.azimuth % 360.
         elevation = id_data.data.elevation
 
         if elevation == 0.:
             # Get telescope position. Will flush data stream in order to get the most updated position
-            position = await self.mcs.tel_mountEncoders.next(flush=True,
-                                                             timeout=self.cmd_timeout)
-            azimuth = position.azimuthCalculatedAngle
-            elevation = position.elevationCalculatedAngle
+            position = await self.ptg.tel_currentTargetStatus.next(flush=True,
+                                                                   timeout=self.cmd_timeout)
+            # These values comes as +DD:MM:SS.SS strings from the pointing component. Need to parse them
+            # to floats here.
+            # wrapping azimuth between 0 and 360, ataos dont care about wrap.
+            azimuth = str_to_float(position.demandAz) % 360.
```

Comment:
I suggest instead:
```
from astropy.coordinates import Angle
import astropy.units as u
...
azimuth = Angle(position.demandAz, u.deg).wrap_to(Angle(360, u.deg)).deg
elevation = Angle(position.demandEl, u.deg).deg
```
where the final `.deg` turns them into floats.

It is also worth considering whether you want to keep these values as `Angle`s. But that would likely require additional changes and I'm not sure what Angle properties would be useful enough to justify that.